EchoPro/computation/spatial.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+import geopy.distance
+
+def calculate_bounds( group ,
+                      contrast ):
+    """
+    Calculates latitude/longitude boundary box    
```

Comment:
It is not a box but a line?

Reply:
Addressed in Commit [22bf7e1](https://github.com/uw-echospace/EchoPro/pull/176/commits/22bf7e14e654624647a38f26d0749feeff7013a5).

Reply:
Oh actually my comment was regarding the docstring, not the function name.
```suggestion
    Calculates start and end latitude/longitude
```
You may want to make similar changes to the Notes.